modules/mobile/docs/schemas/ClaimsAndAppealsOverviewAppealsError.yml

```diff
@@ -0,0 +1,107 @@
+type: object
+required:
+  - data
+  - meta
+properties:
+  data:
+    type: array
+    items:
+      type: object
+      properties:
+        id:
+          type: string
+          example: "600117255"
+          description: Claim Id
+        type:
+          type: string
+          example: "claim"
+          enum:
+            - claim
+        attributes:
+          type: object
+          required:
+            - subtype
+            - completed
+            - dateFiled
+            - updatedAt
+            - displayTitle
+          properties:
+            subtype:
+              type: string
+              example: "Compensation"
+              description: Human readable string for claim or appeal subtype
+            completed:
+              type: boolean
+            dateFiled:
+              type: string
+              format: date
+              example: "2020-01-01"
+            updatedAt:
+              type: string
+              format: date
+              example: "2020-01-01"
+            displayTitle:
+              type: string
+              description: Formatted title for display in mobile overview list
+              example: "disability compensation appeal"
+  meta:
+    properties:
+      errors:
+        type: array
+        description: Array of objects of failing upstream services. Used for debugging only.
+        items:
+          type: object
+          required:
+            - service
+            - errorDetails
+          properties:
+            service:
+              type: string
+              example: "appeals"
+              enum:
+                - appeals
+            errorDetails:
+              type: object
```

Comment:
Looking at the example I pulled from the specs, it looks like errorDetails is an array of objects.

Reply:
Ah you're right, thanks for catching that